packages/zoe/src/state.js

```diff
@@ -155,33 +154,24 @@ const makeIssuerTable = () => {
             ]).then(([brand, mathHelpersName, purse]) => {
               const amountMath = makeAmountMath(brand, mathHelpersName);
               const issuerRecord = {
-                issuer,
                 brand,
+                issuer,
                 purse,
                 amountMath,
               };
-              brandToIssuerMap.init(brand, issuer);
-              table.create(issuerRecord, issuer);
+              table.create(issuerRecord, brand);
+              issuerToBrand.init(issuer, brand);
               issuersInProgress.delete(issuer);
-              return table.get(issuer);
+              return table.get(brand);
             });
             issuersInProgress.init(issuer, synchronousRecordP);
             return synchronousRecordP;
           }
-          return table.get(issuer);
         });
       },
       getPromiseForIssuerRecords: issuerPs =>
         Promise.all(issuerPs.map(customMethods.getPromiseForIssuerRecord)),
-      getIssuerRecordByBrand: brand => {
-        const issuer = brandToIssuerMap.get(brand);
-        return table.get(issuer);
-      },
-      getAmountMathByBrand: brand => {
-        const issuer = brandToIssuerMap.get(brand);
-        const { amountMath } = table.get(issuer);
-        return amountMath;
-      },
+      brandFromIssuer: issuer => table.get(issuerToBrand.get(issuer)).brand,
```

Comment:
is there a reason to go through the issuer table?
```suggestion
      brandFromIssuer: issuerToBrand.get,
```